redis/src/aio/connection_manager.rs

```diff
@@ -316,10 +315,13 @@ impl ConnectionManager {
         number_of_retries: usize,
         connection_config: &AsyncConnectionConfig,
     ) -> RedisResult<MultiplexedConnection> {
-        let retry_strategy = exponential_backoff.map(jitter).take(number_of_retries);
+        let retry_strategy = exponential_backoff.take(number_of_retries);
```

Comment:
why did you remove the jitter?

Reply:
No specific reason, for my team the jitter was an issue and we made it into an optional feature. I can add it back if was a desired behaviour

Reply:
Please do.

Reply:
[917ea10](https://github.com/redis-rs/redis-rs/pull/1334/commits/917ea1042b7fad9363210a4032ddd079c5efe18e)